gym/vector/vector_env.py

```diff
@@ -211,6 +213,27 @@ def seed(self, seed=None):
             "Please use `env.reset(seed=seed) instead in VectorEnvs."
         )
 
+    def _add_info(self, infos: dict, info: dict, env_num: int):
```

Comment:
While a docstring here is not necessary, this is a complex and important functions. Could you give it a docstring to explain what you are doing? And a return type

Technically, we don't need the return type as infos is a reference here so it will be updated outside the function. However, this is not a critical problem and may make more sense to return it